Vagrantfile

```diff
@@ -38,12 +38,13 @@ Vagrant.configure(2) do |config|
 			privileged: false,
 			path: './scripts/vagrant-linux-unpriv-bootstrap.sh'
 
-        # Expose the nomad api and ui to the host
-        vmCfg.vm.network "forwarded_port", guest: 4646, host: 4646, auto_correct: true
+		# Expose the nomad api and ui to the host
+		vmCfg.vm.network :forwarded_port, guest: 4646, host: 4646, auto_correct: true
+		vmCfg.vm.network :forwarded_port, guest: 8500, host: 8500, auto_correct: true
 
-        # Expose Ember ports to the host (one for the site, one for livereload)
```

Comment:
Note to reviewers: the reason for all the whitespace diffs here is that this file is done with tabs (for some reason it seems many of our scripts are?), whereas these few lines slipped in with spaces.